msamp/nn/__init__.py

```diff
@@ -7,6 +7,8 @@
 from msamp.nn.module import ScalingModule
 from msamp.nn.state import model_state
 from msamp.nn.linear import FP8Linear, LinearReplacer
+from msamp.nn import functional
 from msamp.nn.clip_grad import clip_grad_norm_
+del functional
```

Comment:
For torch, `torch.nn.functional` equals to `torch.functional`.
How about just exporting `functional` to let msamp can call functional by both `msamp.functional` and `msamp.nn.functional`?

Reply:
Great. I will change it.

Reply:
OK, looks they just have few common attributes. Just keep `del functional`